tests/php7/methods/isPositive.phpt

```diff
@@ -0,0 +1,55 @@
+--TEST--
+Decimal::isPositive
+--SKIPIF--
+<?php
+if (!extension_loaded("decimal")) echo "skip";
+?>
+--FILE--
+<?php
+use Decimal\Decimal;
+
+/**
+ * Shortcut to construct a new decimal.
+ */
+function decimal(...$args) { return new Decimal(...$args); }
+
+$tests = [
+    /* Number, expected */
+
+    [ "1E-50",  true],
+    ["-1E-50",  false],
+
+    ["0",       false],
+    ["-0",      false],
+    [1,         true],
+    [2,         true],
+    [3,         true],
+
+    [-1,        false],
+    [-2,        false],
+    [-3,        false],
+
+    ["1.5",     true],
+    ["2.5",     true],
+    ["3.5",     true],
+
+    ["-1.5",    false],
+    ["-2.5",    false],
+    ["-3.5",    false],
+
+    [ "NAN",    true],
```

Comment:
Not sure this is the expected behavior.

Reply:
I think NaN is neither positive nor negative.

Reply:
Fixed in #68.